src/assignmentB/AssignmentB02.java

```diff
@@ -1,5 +1,96 @@
 package assignmentB;
 
-public class AssignmentB02
+import javax.swing.*;
+import javax.swing.table.DefaultTableModel;
+import java.awt.*;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Locale;
+
+public class AssignmentB02 extends JFrame
 {
+    Calendar cal = new GregorianCalendar();
+    DefaultTableModel model;
+    JLabel label;
+
+    //TODO: add text box to add/view appointments
+
+    AssignmentB02()
+    {
+        this.setTitle("Appointment Calendar");
+        this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        this.setSize(640, 480);
+        this.setVisible(true);
+
+        //TODO: fix formatting... need fixed table that doesn't resize with window
+        // BoarderLayout() ??
+        this.setLayout(new FlowLayout());
+
+        label = new JLabel();
+        label.setHorizontalAlignment(SwingConstants.CENTER);
+
+        JButton prevMonth = new JButton("Prev");
```

Comment:
A more descriptive name could be `prevMonthButton`.

Reply:
name updated to prevMonthButton